.github/workflows/pytest-dbutils-influxdbfuncts.yml

```diff
@@ -1,30 +1,29 @@
 ---
 # This workflow is meant as a foundational workflow for running integration/unit tests on the
-# platform.  For this workflow we are testing the
-#
-# volttrontesting/testutils directory using pytest.
-#
+# platform.
 # This workflow also shows the caching mechanisms available for storage
 # and retrieval of cache for quicker setup of test environments.
 
-name: Testing dbutils directory
+
+name: Testing influxdbutils
 on: [pull_request]
 
 jobs:
   build:
+    env:
+      TEST_FILE: influxdbutils.py
```

Comment:
I think this should be the whole filename....you will confuse people (me included) without the full file.